test/Makefile

```diff
@@ -179,6 +179,15 @@ setup_spiderpool:
 		fi ; \
 		if [ "$(E2E_SPIDERPOOL_ENABLE_COORDINATOR)" == "true" ] ; then \
 		    HELM_OPTION+=" --set coordinator.enabled=true " ; \
+			if [ "$(INSTALL_CILIUM)" == "true" ] ; then \
```

Comment:
Actually, This is the same problem as https://github.com/spidernet-io/spiderpool/issues/2433. It was fixed by https://github.com/spidernet-io/spiderpool/pull/2434.  it's OK to merge this part of the code for me. But we should revert these changes until https://github.com/spidernet-io/spiderpool/pull/2434 has been merged.

Reply:
ok